lib/game/updateUniverse.js

```diff
@@ -0,0 +1,53 @@
+// Includes
+const http = require('../util/http.js').func
+const getGeneralToken = require('../util/getGeneralToken.js').func
+
+// Args
+exports.required = ['universeId', 'settings']
+exports.optional = ['jar']
+
+// Docs
+/**
+ * 🔐 Modifies a universe's settings
+ * @category Game
+ * @alias updateUniverse
+ * @param {number} universeId - The universeId of the experience
+ * @param {UniverseSettings} settings - The settings to update
+ * @returns {Promise<UpdateUniverseResponse>}
+ * @example const noblox = require("noblox.js")
+ * // Login using your cookie
+ * noblox.updateUniverse(2421261122, { name: "The best game ever!" })
+**/
+
+// Define
+function updateUniverse (universeId, settings, jar, token) {
+  return new Promise((resolve, reject) => {
+    return http({
```

Comment:
This falls into the anti-pattern frequently present within the library - The http promise is not caught, as it is within a Promise which has been manually created. Please add .catch to this promise!